src/templates/base/2019/contributors.html

```diff
@@ -262,27 +271,37 @@ <h1 class="title title-lg">
   </div>
   <div class="contributors {{ ' '.join(config.teams.keys()) }}">
```

Comment:
What do you think about using a `form` element here, or using `role="form"`? I am thinking this would expose the filtering functionality as a landmark and make it potentially easier to discover.

Reply:
Will have a look in follow up PR next week.